packages/components/src/components/link/_link.scss

```diff
@@ -68,6 +64,30 @@
     font-weight: 400;
     cursor: not-allowed;
   }
+
+  .#{$prefix}--link.#{$prefix}--link--inline {
+    text-decoration: underline;
+
+    &:hover {
+      color: $hover-primary-text;
+    }
+
+    &:focus {
+      text-decoration: none;
+    }
+
+    &:visited {
```

Comment:
Do we need to define this behavior a second time? since it's covered by the regular link class now?

Reply:
Good question, without the `color` rule in this ruleset, the color is overriden by original color of `.bx--link` (`$link-01`).